src/main/kotlin/com/autonomousapps/model/intermediates/Reason.kt

```diff
@@ -104,6 +104,16 @@ internal sealed class Reason(open val reason: String) {
     override val configurationName: String = "implementation"
   }
 
+  @TypeLabel("testInstrumentationRunner")
+  @JsonClass(generateAdapter = false)
+  data class TestInstrumentationRunner(override val reason: String) : Reason(reason) {
+    init {
+      buildReason(setOf(reason), "Declares", Kind.AndroidTestInstrumentationRunner)
+    }
+
+    override val configurationName: String = "runtimeOnly"
```

Comment:
Open question if this should be runtimeOnly or implementation, but I think this makes sense.